recipes/default.rb

```diff
@@ -17,6 +17,7 @@
 # limitations under the License.
 #
 
+
```

Comment:
can you drop this line? I'm pretty sure it was a mistake? 👍

Reply:
Sure ,